services/src/test-library/src/main/java/org/openkilda/testing/service/northbound/NorthboundServiceImpl.java

```diff
@@ -312,7 +312,7 @@ private IslInfoData convertToIslInfoData(LinkDto dto) {
     private SwitchInfoData convertToSwitchInfoData(SwitchDto dto) {
         return new SwitchInfoData(
                 new SwitchId(dto.getSwitchId()),
-                SwitchState.from(dto.getState()),
+                SwitchState.valueOf(dto.getState()),
```

Comment:
How do we recoup ability to accept "state" in lower/upper case?

Reply:
W/o the method "from" it won't be possible. I'm reverting back this change.

Reply:
I don't like the method "from" either, that's why we consider migrating to upper-case in enums.